### PR TITLE
MNT: Re-rendered with conda-smithy 2.4.5

### PR DESCRIPTION
**Pull request by tschoonj**
Also: build packages compatible with multiple numpy versions

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,14 +9,14 @@ source:
...
-  number: 0
+  number: 1
 
 requirements:
   build:
     - python
     - toolchain
     - setuptools
-    - numpy x.x
+    - numpy 1.10.x
```
Comment (ericdill) [inline recipe/meta.yaml]: I think that should be 1.10.*

Comment (tschoonj) [reply recipe/meta.yaml]: Fixed!

**Comment by ericdill** [comment]
There's weird dependencies with numpy and windows and python 3.5-3.6. See [here](https://conda-forge.org/docs/meta.html#building-against-numpy)

tl;dr I think you need numpy 1.11 for py36

```yaml
build:
  - numpy 1.8.*  # [not (win and (py35 or py36))]
  - numpy 1.9.*  # [win and py35]
  - numpy 1.11.*  # [win and py36]
run:
  - numpy >=1.8  # [not (win and (py35 or py36))]
  - numpy >=1.9  # [win and py35]
  - numpy >=1.11  # [win and py36]
```

**Comment by tschoonj** [comment]
Finally all green... ready to merge?

**Comment by ericdill** [comment]
lgtm!

**Comment by tschoonj** [comment]
Thanks!

**Pull request merged**